include/internal/catch_common.hpp

```diff
@@ -22,7 +24,7 @@ namespace Catch {
         return s.find( infix ) != std::string::npos;
     }
     void toLowerInPlace( std::string& s ) {
-        std::transform( s.begin(), s.end(), s.begin(), ::tolower );
+        std::transform( s.begin(), s.end(), s.begin(), static_cast<int(*)(int)>(std::tolower) );
```

Comment:
Another reasonable approach might be:
```
int (*tolowep)(int) = std::tolower;
std::transform( s.begin(), s.end(), s.begin(), tolowerp );
```
That will select the correct overload, but avoids actually writing `static_cast`.